### DISCUSSION TITLE
Adjusting the digital ROI margin

### DISCUSSION THREAD

**Discussion by DasEiserneO** [jomjol/AI-on-the-edge-device#3178] [Q&A]
Hi,
I have problem with the recognition of some numbers with the dig-class11 model.

![Bildschirmfoto 2024-08-12 um 15 52 03](https://github.com/user-attachments/assets/bdb8138d-b8c9-4b38-8db1-238d1d62382f)

As you can see in the picture, there is a frame around the last three numbers.
The problem is that the model also analyzes this edge for some numbers because it is in the edge area of the ROI.
e.g. 4 is recognized by the model as 9
However, if a number contains an "upper element" (I hope you understand what I mean) everything works fine
e.g. 3 is recognized by the model as 3

Is there a possibility to change the border of a ROI?
Or does anyone have another idea?

PS: I've already tried simply taping it off, but then I have the same problem with the shadow of the tape, as I can't stick it directly to the display.

Thanks in advance

**Comment by SybexX** [comment]
Post a picture of the "Digital ROI" page so that people can see how you created the ROIs and then they might give you some tips.

https://github.com/jomjol/AI-on-the-edge-device/discussions/2500#discussioncomment-10299430

**Comment by DasEiserneO** [comment]
Here is a picture of the ROI settings I have tried.
![Bildschirmfoto 2024-08-12 um 18 23 47](https://github.com/user-attachments/assets/c82768cf-7f3a-4b62-914b-f619db53f37f)
![Bildschirmfoto 2024-08-12 um 18 23 59](https://github.com/user-attachments/assets/8706378e-9c4b-4736-9b6a-2ccb6e47f7ae)

1) recognizes a 4 as 1
2) recognizes a 4 as 9

**Comment by SybexX** [comment]
you could try the current rolling version and zoom in a bit with the revised zoom function to increase the size of the ROIs: https://github.com/jomjol/AI-on-the-edge-device/actions/runs/10310723944
I would also play around with the camera settings to improve the quality of the image, especially increasing the brightness and improving the contrast.
this could also help: https://jomjol.github.io/AI-on-the-edge-device-docs/Reference-Image/#focus

**Reply by caco3** [reply]
another tip is to rotate the image a bit CCW to work against the tilted digits.
